packages/apps/shopify-app-remix/src/server/authenticate/admin/scope/request.ts

```diff
@@ -4,7 +4,7 @@ import type {BasicParams} from '../../../types';
 import {redirectToInstallPage} from '../helpers/redirect-to-install-page';
 import {AdminApiContext} from '../../../clients';
 
-import {fetchScopeInformation} from './client/fetch-scopes-information';
+import {fetchScopeDetail} from './client/fetch-scopes-details';
 
 export function requestScopesFactory(
```

Comment:
The return value for `request` wasn't updated since this implementation simply redirects to the install page.